bedrock/newsletter/templates/newsletter/management.html

```diff
@@ -72,6 +72,8 @@ <h1>{{ ftl('newsletters-manage-your-email-preferences') }}</h1>
       data-recovery-url="{{ recovery_url }}"
       data-unsubscribe-url="{{ unsubscribe_url }}">
 
+      <input type="hidden" name="source_url" value="{{ request.build_absolute_uri() }}">
```

Comment:
This renders the URL complete with token. I'm guessing we want to include only the `/newsletter/existing/` path?

Reply:
Yes, we don't want the token. On my local testing it didn't include the token. I can pass in the `request.path` here, which doesn't include the query string params, and it will build the URL based on that, rather than based on `get_full_path()` which it defaults to.